dials.go

```diff
@@ -24,6 +24,16 @@ type Params struct {
 	//  - a Verify() method fails after re-stacking when a new version is
 	//    provided by a watching source
 	OnWatchedError WatchedErrorHandler
+
+	// SkipInitialVerification skips the initial call to `Verify()` on any
+	// configurations that implement the `VerifiedConfig` interface.
+	//
+	// In cases where later updates from Watching sources are depended upon to
+	// provide a configuration that will be allowed by Verify(), one should set
+	// this to true.  `sourcewrap.Blank` is one such example where values are
+	// not initially available and would not be able to be considered for
+	// initial verification.
```

Comment:
I feel like this part about `sourcewrap.Blank` is a bit misleading. It implies that one would *always* want to set this when using the blank source.
Perhaps this reference to `sourcewrap.Blank` should spell out the whole `ez`/file-source-insertion use-case?